e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -141,6 +141,7 @@ export function openQuestionsSidebar() {
 
 export function createNewTab() {
   cy.findByLabelText("Create new tab").click();
+  cy.findByText(`There's nothing here, yet.`).should("be.visible");
```

Comment:
Do we want this assertion here in a helper that can be used in different scenarios? I ask because if we add a tab to a dashboard that currently has no cards, this check will fail. It is an unlikely scenario - but it could be confusing.

Reply:
Good point, I'll move this into the test body